frontend/src/Admin/components/ShowSubmissionsTab.js

```diff
@@ -1,5 +1,238 @@
-import React from 'react'
+import React, { Component, Fragment } from 'react'
+import { Alert } from 'reactstrap'
+import ReactTable from 'react-table'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import FaYouTube from 'react-icons/lib/fa/youtube'
+import FaVimeo from 'react-icons/lib/fa/vimeo'
+import FaStar from 'react-icons/lib/fa/star'
+import FaStarOpen from 'react-icons/lib/fa/star-o'
+import FaBook from 'react-icons/lib/fa/book'
 
-export default function () {
-  return 'Submissions'
+import { getImageThumbnail } from '../../utils'
+
+// Alert Types
+const SUCCESS = 'SUCCESS'
+const ERROR = 'ERROR'
+
+const PhotoThumbnail = styled.img`
+  height: auto;
+  max-height: 5em;
+  max-width: 100%;
+  min-width: 3em;
+  width: auto;
+`
+class ShowSubmissionsTab extends Component {
+  static propTypes = {
+    updateInvite: PropTypes.func.isRequired,
+    show: PropTypes.shape({
+      entries: PropTypes.arrayOf(
+        PropTypes.shape({
+          id: PropTypes.string.isRequired,
+          title: PropTypes.string.isRequired,
+          entryType: PropTypes.oneOf(['PHOTO', 'VIDEO', 'OTHER']).isRequired,
+          invited: PropTypes.bool,
+          path: PropTypes.string,
+          provider: PropTypes.oneOf(['youtube', 'vimeo']),
+          videoId: PropTypes.string
+        })
+      )
+    }).isRequired
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      alertVisible: true,
+      alertType: ''
+    }
+  }
+
+  onDismiss = () => {
+    this.setState({
+      alertVisible: false,
+      alertType: ''
+    })
+  }
+
+  updateShowInvite = (id, value) => {
+    const { updateInvite } = this.props
+    updateInvite(id, value)
+      .then(() => {
+        this.setState({
```

Comment:
I don't know if showing alerts is useful here since the star icon will have changed color signaling a change.

Reply:
we had the alert in the original story. I say we keep it for now. We can remove it later if Nanette doesn't like it.

Reply:
An alert is not a toast, an imo adds too much clutter to the page. We can keep it in for now and add it to our tech debt list.